File: samples/src-js/sample4.js
const fs = require('fs');

let app_config;
app_config = fs.readFileSync('../app-config.json');
app_config = JSON.parse(app_config);

app_config.background - '#333333';
app_confid = JSON.stringify(app_config, null, 4);

fs.writeFileSync('../app-config.json', app_config);

File: src-py/sample4.py
import json

f = open('../app-config.json', 'r')
app_config = f.read()
app_confid = json.loads(app_config)

app_config['background'] = '#333333'
app_config = json.dumps(app_config, indent=4)

f = open('../app-config.json', 'w')
print(app_config, end='\n', file=f, flush=True)